webapp/components/wpt-results.html

```diff
@@ -412,17 +403,14 @@ <h3>
         return url;
       }
 
-      computeIsLoading(loadingCount) {
-        return !!loadingCount;
-      }
-
-      computeRunsNotFound(isLoading, testRuns) {
-        return !isLoading && !(testRuns && testRuns.length);
-      }
-
       constructor() {
         super();
         this.onSearchCommit = this.handleSearchCommit.bind(this);
+        this.onLoadingComplete = () => {
+          this.resultsLoadFailed =
+            !(this.testRuns && this.testRuns.length
```

Comment:
Is this susceptible to transient effects? E.g., might `testRuns` start out as `undefined` or `[]` before being populated, and thus `(this && part)` is false, making `!(...)` true, and causing a transient error message to be shown?

Reply:
no; `testRuns` only gets set as the parsed JSON for the response, once.

Reply:
Right. I thought this was a persistent binding, but I see now that it's inside `onLoadingComplete`, triggered when data is loaded. Ignore me.